src/pixelator/report/qcreport/builder.py

```diff
@@ -82,21 +82,21 @@ def _load_template(self) -> Tuple[LxmlElement, LxmlElement]:
         body = document.find("body")
 
         if body is None:
-            raise AssertionError("Could not find body element in webreport template")
+            raise AssertionError("Could not find body element in QC report template")
```

Comment:
IS it easy to test for this so we cover it?

Reply:
This assertion error should basically never happen. That would be an issue with the qc-report html template file.
This not happening is tested with eg: test_write_webreport